packages/flutter/test/widgets/reparent_state_harder_test.dart

```diff
@@ -49,7 +49,7 @@ class OrderSwitcherState extends State<OrderSwitcher> {
 }
 
 class DummyStatefulWidget extends StatefulWidget {
-  const DummyStatefulWidget(Key? key) : super(key: key);
+  const DummyStatefulWidget(super.key);
```

Comment:
Unfortunately we'll probably have to revert this one.

I believe this (along with the `TestWidget` constructors) doesn't vibe with the Dart syntax, since `key` is a named parameter (rather than a positional parameter)—`({super.key})` works fine, but `(super.key)` doesn't.

Reply:
Alternatively, we could change it to a named parameter (not a breaking change since it's just a test), but that would involve more than just changing a single line.

Reply:
> You could either revert or change it to a named parameter (not a breaking change since it's just a test, but it'd involve more than just changing a single line).

I would want to do the second option but I can't `flutter pub get` because of timeout (I don't remember what was the issue but I'm sure) it's due to my slow network connection